docs/semgrep-app/scm.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: scm 
+append_help_link: true
+title: Integrating Semgrep into source code management (SCM) tools
+description: "Integrate Semgrep into self-hosted and custom SCM tools such as GitHub Enterprise and GitLab Self Hosted."
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Integrating Semgrep into source code management (SCM) tools
+
+Semgrep App's Team tier supports repositories hosted on both **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. Create multiple integrations in different self-hosted or custom servers.
+
+## Prerequisites
+
+You need the following permissions to integrate Semgrep into GHE or GLSM servers:
+
+* Permission to create a personal access token (PAT) for the repository to scan Semgrep on. This PAT is used to enable Semgrep App to create pull request (PR) or merge request (MR) comments for findings that can be [autofixed](../notifications/#automatically-fix-your-findings-through-pull-or-merge-requests).
+* Permission to add CI/CD secrets into your [GitHub](https://docs.github.com/en/actions/security-guides/encrypted-secrets) or [GitLab](https://docs.gitlab.com/ee/ci/secrets/) environments.
+
+## Integrating Semgrep into GitHub Enterprise or GitLab Self Managed 
+
+Integrate Semgrep into these custom source code management (SCM) tools by following the steps below:
+
+1. Sign in to [Semgrep App](https://semgrep.dev/login).
+2. Click **[Settings](https://semgrep.dev/orgs/-/settings)** > **Create new parent organization**.
+
+<div class = "bordered">
+
+![Screenshot of settings for parent organization](../img/app-parent-org.png "Screenshot of settings for parent organization")
+
+</div>
+
+3. Enter the name of your parent organization.
+4. Sign out, then sign in again to enable SCM management.
+5. Click **Settings** > **SCM management** > **Configure new SCM**.
+
+<div class = "bordered">
+
+![Screenshot of SCM configuration tab](../img/app-scm.png)
+
+</div>
+
+6. Select your SCM provider.
+7. For **GitHub Enterprise**, follow these steps:
+
+    1. Create a personal access token (PAT) by following the steps outlined in this [guide to creating a PAT](https://docs.github.com/en/enterprise-server@3.1/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).

```

Comment:
At this stage, does the user need to know what permissions they need to enable? I recall users asking this during onboarding.

Reply:
👍

Reply:
Having an explanation for the required scopes would be nice too

Reply:
I believe this is covered in the prerequisites section above. @dlukeomalley 

For the scopes @zyannes I'll PM you to double check :)